HousingFinanceInterimApi/V1/UseCase/GenerateRentPositionUseCase.cs

```diff
@@ -71,28 +70,57 @@ public async Task<StepResponse> ExecuteAsync()
                     LoggingHandler.LogInfo($"Folder ID: {googleFileSetting.GoogleIdentifier}");
                     LoggingHandler.LogInfo($"File count: {folderFiles.Count}");
 
-                    LoggingHandler.LogInfo($"Deleting old files");
-                    foreach (var file in folderFiles.Where(f => f.Name.Equals(fileName)).ToList())
-                    {
-                        await _googleClientService.DeleteFileInDrive(file.Id).ConfigureAwait(false);
-                    }
-
                     var isSuccess = await _googleClientService.UploadCsvFile(rentPosition, fileName, googleFileSetting.GoogleIdentifier)
                         .ConfigureAwait(false);
 
                     if (!isSuccess)
                         throw new Exception("Failed to upload to Rent Position folder (Qlik)");
 
+                    LoggingHandler.LogInfo($"Deleting old files");
+                    foreach (var file in folderFiles.Where(f => f.Name.Equals(fileName)).ToList())
+                    {
+                        await _googleClientService.DeleteFileInDrive(file.Id).ConfigureAwait(false);
+                    }
```

Comment:
The
``` C#
var fileName = "RentPosition.csv";
```
Is that how `folderFiles` are named? I thought they'd be prefixed/suffixed with date strings :thinking: Could be wrong though.

Reply:
All that was changed for this part was moving the exception up. In the Rent Position (Qlik) folder the file is named `RentPosition.csv`. It's in the `HFS Production / Rent Position`` aka Rent Position Bkp folder where it's prefixed with a date stamp

Reply:
Reverted in https://github.com/LBHackney-IT/housing-finance-interim-api/pull/103/commits/6c46a9ad9afe5b1d0bf2e6e9221896d01f70a996